packages/igx-templates/igx-ts/projects/_base/files/package.json

```
@@ -32,9 +32,9 @@
   },
   "devDependencies": {
     "@angular-devkit/build-angular": "0.900.0-rc.7",
```

The `@angular-devkit` dep should also be non-strict
```suggestion
    "@angular-devkit/build-angular": "~0.900.0-rc.0",
```